notebooks/242-freevc/242-freevc.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    cmodel = get_cmodel()
</pre><p>how about naming model object like that <code>encoder_model </code> instead of <code>smodel</code> ？which may be more clear for our users    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
These are the original names from the FreeVC project. Do you think it makes sense to change them?